.travis.yml

```diff
@@ -1,22 +1,23 @@
+dist: bionic
 language: python
 python:
   - "3.8"
 
-# The pipeline requires GNU parallel; Python retry and requests modules; and bcftools.
-before_install:
-  - sudo apt-get -y install parallel
+# The pipeline requires GNU parallel; bcftools; and certain Python modules
 install:
-  - pip -q install retry requests
-  - git clone -q git://github.com/samtools/htslib.git
-  - git clone -q git://github.com/samtools/bcftools.git
-  - cd bcftools && make --quiet --jobs `nproc` && cd ..
-  - export PATH=$PATH:bcftools
+  - sudo apt update
+  - sudo apt -y install samtools bcftools parallel libbz2-dev liblzma-dev
+  - pip -q install -r requirements.txt
```

Comment:
Why the "-q" switch?

Reply:
`-q` only affects info-level messages, which we don't want. If anything does go wrong, it'll report it (and also throw a non-zero return code), for example:

```
$ pip3 -q install nonexistent_package
ERROR: Could not find a version that satisfies the requirement nonexistent_package (from versions: none)
ERROR: No matching distribution found for nonexistent_package
$ echo $?
1
```